EIPS/eip-unified_documentation.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: <to be assigned>
+title: Unified Documentation
+author: Micah Dameron(@chronaeon)
+Discussions-to: https://github.com/ethereum/EIPs/issues/1054 
+status: Draft
+type: Administrative
```

Comment:
'Administrative' is not a valid type, and 'Emergent' is not a valid category. Please choose a valid type (and if appropriate, category) as specified in EIP-1.